content/developers/api-reference/events-api/index.md

```diff
@@ -86,6 +86,174 @@ The response is:
   "transaction_id": "0x07569"
 }
 ```
+
+### Document Profile Event Creation
+There are two [Document Profile Events](/developers/developer-patterns/document-profile/) that are available as part of the document lifecycle. These are to `publish` a new version and to `withdraw` the document from use.
+
+#### Publish
+Define the Event parameters and store in `/path/to/jsonfile`:
+
+```json
+{
+    "behaviour": "RecordEvidence",
+    "operation": "Record",
+    "asset_attributes": {
+        "document_hash_value":"xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
+        "document_hash_alg":"sha256",
+        "document_status": "Published",
+        "document_version":"2"
+    },
+    "event_attributes": {
+        "arc_description":"Publish version 2 of Test Document",
+        "arc_display_type":"Publish",
+        "document_version_authors": [
+                        {
+                "display_name": "George",
+                "email": "george@rainbow.tv"
+            },
+            {
+                "display_name": "Zippy",
+                "email": "zippy@rainbow.tv"
+            },
+            {
+                "display_name": "Bungle",
+                "email": "bungle@rainbow.tv"
+            }
+        ]
+    }
+}
+```
+
+Add the request to the Asset record by POSTing it to the resource:
+
+```bash
+curl -v -X POST \
+    -H "@$BEARER_TOKEN_FILE" \
+    -H "Content-type: application/json" \
+    -d "@/path/to/jsonfile" \
+    https://app.rkvst.io/archivist/v2/assets/b2c68bd2-1274-4f76-b12f-8b5d36163f4f/events
+```
+
+The response is:
+
+```json
+{
+    "identity": "assets/b2c68bd2-1274-4f76-b12f-8b5d36163f4f/events/1cdfe161-7f7f-4455-9c8b-b6ddc81c6202",
```

Comment:
`"assets/xxxxxxxx-xxxx-xxxx-xxxxxxxxxxxxxxxxx/events/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",`

Reply:
I kept the asset ID in clear text because it matches the one that was created in the Assets API reference so that you can see the link.

Also these examples were generated on Soak not Prod. Do they need to be hidden? We haven't hidden the asset and event IDs in any of the other articles.